src/status_im2/contexts/shell/activity_center/notification/reply/view.cljs

```diff
@@ -106,4 +112,4 @@
 
                                                       :else
                                                       nil)
-                              :body                 (get-message-content message)}}]]]))
+                              :body                 (get-message-content message album-messages)}}]]]))
```

Comment:
pass (rf/sub [:mediaserver/port]) here as parameter

Reply:
Done, Thank you